ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-cpou-upgrade-from-4.10.yaml

```diff
@@ -47,6 +47,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.10"
+  source:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.11"
```

Comment:
```suggestion
      version: "4.10"
```

Reply:
same to https://github.com/openshift/release/pull/57024/files#r1772434496